tests/integ/data/schema-no-tagging-permission.json

```diff
@@ -0,0 +1,189 @@
+                }
+            },
+            "required": [
+                "Key",
+                "Value"
+            ],
+            "additionalProperties": false
+        }
+    },
+    "properties": {
+        "Arn": {
+            "description": "",
+            "type": "string"
+        },
+        "AssumeRolePolicyDocument": {
+            "description": "The trust policy that is associated with this role. Trust policies define which entities can assume the role. You can associate only one trust policy with a role. For an example of a policy that can be used to assume a role, see [Template Examples](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#aws-resource-iam-role--examples). For more information about the elements that you can use in an IAM policy, see [Policy Elements Reference](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements.html) in the *User Guide*.",
+            "type": [
+                "object",
+                "string"
+            ]
+        },
+        "Description": {
+            "description": "A description of the role that you provide.",
+            "type": "string"
+        },
+        "ManagedPolicyArns": {
+            "description": "A list of Amazon Resource Names (ARNs) of the IAM managed policies that you want to attach to the role.\n For more information about ARNs, see [Amazon Resource Names (ARNs) and Service Namespaces](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) in the *General Reference*.",
+            "type": "array",
+            "uniqueItems": true,
+            "insertionOrder": false,
+            "items": {
+                "type": "string"
+            }
+        },
+        "MaxSessionDuration": {
+            "description": "The maximum session duration (in seconds) that you want to set for the specified role. If you do not specify a value for this setting, the default value of one hour is applied. This setting can have a value from 1 hour to 12 hours.\n Anyone who assumes the role from the CLI or API can use the ``DurationSeconds`` API parameter or the ``duration-seconds`` CLI parameter to request a longer session. The ``MaxSessionDuration`` setting determines the maximum duration that can be requested using the ``DurationSeconds`` parameter. If users don't specify a value for the ``DurationSeconds`` parameter, their security credentials are valid for one hour by default. This applies when you use the ``AssumeRole*`` API operations or the ``assume-role*`` CLI operations but does not apply when you use those operations to create a console URL. For more information, see [Using IAM roles](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_use.html) in the *IAM User Guide*.",
+            "type": "integer"
```

Comment:
you dont need a full schema. Just like that would be enough:
```
{
    "tagging": {
        "taggable": true,
        "tagOnCreate": true,
        "tagUpdatable": true,
        "cloudFormationSystemTags": false,
        "tagProperty": "/properties/Tags"
    }
}
```

Reply:
Simplified the json file to include only permission related stuff